src/style/sections/exports/exports.less

```diff
@@ -165,6 +165,33 @@
     flex-direction: row;
 }
 
+.layer-exports__header {
+   width: 100%;
+
+   .formline.formline__space-between {
+        & > div {
+            border-bottom: @hairline solid @underlines;
+            
+            & > input[type="text"] {
```

Comment:
Do you need the `>` selector? Best to have less specific selectors when possible